seekinternational/notification-library
# Email feature

What does this PR do?
---------------------
Adds email feature
- define cc, bcc, to
- define message body like text, html

Adds local dev execution for email

How can I test this?
--------------------
1. Input credentials in scripts/local-email.js
2. Execute script by running `npm run dev:email`

Notes
-----
Do take note that you define either html or text. If you define both then email will take precedence

Input for send email are as follows:
destination: {
    to: [< email address >],
    cc,: [< email address >],
    bcc: [< email address >]
}
from: < source email >,
subject: < subject >,
html: < email template >
text: < text body >
